tests/automation_framework/src/libs/test_base.py

```diff
@@ -155,13 +220,26 @@ def setup_and_build_request_worker_status(self, input_file):
         return 0
 
     def getresult(self, output_obj, submit_response={}):
+        """
```

Comment:
Mention that in case of wo_submit negative tests the error code won't be 5 and hence wo_get result won't be called.